src/pages/VideoDetail/VideoDetail.page.jsx

```diff
@@ -16,26 +15,76 @@ import {
   StyledWrapperTitle,
   StyledWrapperIcon,
 } from './VideoDetail.styles.jsx';
+import { storageFavVideos } from '../../utils/storage';
 
```

Comment:
Since the information for this page depends on navigating from Home, one cannot enter directly, or reload the page without losing some information. This may be also the reason why if we navigate to a related video, the section for Related Video List is no longer displayed.
This could be fix by running the fetch request when entering this page too, in case of not having the full video information. For these we could use the request for a specific video instead of a search request.
For this suggestion it would be a good idea to create a hook for this specific use case